tests/testthat/test-tab_style.R

```diff
@@ -429,10 +427,7 @@ test_that("a gt table can store the correct style statements", {
 
   # Check that `tab_row_group(others_label = ...)` still works but
   # issues a warning
-  expect_equivalent(
-    expect_warning(data %>% tab_row_group(others_label = "Others1")),
-    data %>% tab_options(row_group.default_label = "Others1")
-  )
+  expect_warning(data %>% tab_row_group(others_label = "Others1"))
```

Comment:
The pattern doesn't match the other tests here - I could 100% be wrong but are we missing `expect_equal` or something like that?

Reply:
For this case, we really just want to know that there is a warning from a soft deprecation (on our part) and we don't really care about the result.